### PR TITLE
Added Gulp and related dependencies

### PR DESCRIPTION
**Pull request by AniVerma17**
- Added NPM & Gulp configuration for Sass compilation , JS minification etc.
- Created index.html.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,41 @@
 The code and development of the PyLadies Delhi Chapter website is housed here. 
 
 ## Contributing Guidelines 
-Add Assets such as images and other media in assets folder, follow standard coding practises, naming conventions and the regular document object model. Make issues on GitHub to propose new features, bugs and add details on the website and then make a PR referencing the same.   
+Add Assets such as images and other media in assets folder, follow standard coding practises, naming conventions and the regular document object model. Make issues on GitHub to propose new features, bugs and add details on the website and then make a PR referencing the same. 
+
+## Requirements
+- NodeJS & NPM
```
Comment (vipulgupta2048) [inline README.md]: @AniVerma17  You heard the requirements same as me at the meetup. The site needs to be static. No matter what. Think of it as a "client" problem. We would have to work around it. 
**Or** I am cool with it if you take permission from her first. I don't want any work to go waste here. Good work though.

Comment (vipulgupta2048) [reply README.md]: Do send me a screenshot as a direct message.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>PyLadies Delhi Chapter</title>
+	<link rel="shortcut icon" type="image/x-icon" href="build/imgs/favicon.png">
+	<link rel="apple-touch-icon" type="image/x-icon" href="build/imgs/favicon.png">
+	<!-- SEO tags -->
+	<meta name="description" content="Official website of PyLadies Delhi Chapter">
+	<meta name="keywords" content="PyLadies, PyLadies Delhi, PyLadies Delhi Chapter">
+	<!-- Main CSS file...do not modify. -->
+	<link rel="stylesheet" type="text/css" href="build/main.css" />
```
Comment (vipulgupta2048) [inline index.html]: Where is build/main.css?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```
Comment (vipulgupta2048) [inline index.html]: I don't know if viewing on GitHub is wrong or what. But this doesn't seem in accordance with the DOM style. Do make the necessary edits. :+1:

Comment (vipulgupta2048) [reply index.html]: The format of the entire file, not that particular file.

File: README.md (hunk 1 above)
Comment (AniVerma17) [reply README.md]: The site will be static, this is to compile SCSS(Sass) files to CSS and compress JavaScript code to improve performance.

File: index.html (hunk 2 above)
Comment (AniVerma17) [reply index.html]: Will be created automatically when you'll upload ur SCSS files as instructed (refer to README) and run Gulp on the project.

File: index.html (hunk 3 above)
Comment (AniVerma17) [reply index.html]: This is just the bare code....just to specify the tags to link to build files. Modifications will be done as the work proceeds.

**Comment by vipulgupta2048** [comment]
@utkarsh2102  You forgot an approval comment, LGTM. This is not compulsory by any means but these help in ensuring that you know what you were doing, before pressing the green button and whose neck I have to go after if the website breaks :hatching_chick: :+1: 
Habits go a long way. Try to follow the advice given.

**Pull request merged**